github-for-jira.sd.yml

```diff
@@ -298,12 +314,15 @@ environmentOverrides:
               Value: { "Fn::GetAtt": [ "ALB", "LoadBalancerFullName" ] }
             - Name: TargetGroup
               Value: { "Fn::GetAtt": [ "ALBTargetGroup", "TargetGroupFullName" ] }
-          Threshold: 3
+          Threshold: 5
           Priority: Low
           EvaluationPeriods: 3
           Period: 60
           ComparisonOperator: GreaterThanThreshold
-          Statistic: Maximum
+          Statistic: Average
+        LowSeverityAlarmWhenTooManyELB5xxErrors: null
+        LowSeverityAlarmWhenTooManyBackend5xxErrors: null
+        HighSeverityAlarmWhenTooManyELB5xxErrors: null
```

Comment:
We didn't get any ELB alerts surprisingly, but these alerts are also default micros alerts and works for both ALB and ELB (according to the docs). Se wouldn't be wrong to disable them as well.